contributing.md

```diff
@@ -0,0 +1,164 @@
+
+## Add a forum topic
+
+In the [Percona Product Documentation category](https://forums.percona.com/c/percona-product-documentation/71) in the Percona Community Forum, select **New Topic**. Complete the form and select **Create Topic** to add the topic to the forum.
+
+<img src= "./docs/_images/new-topic.png" alt ="create a topic" width = 40% height = 40% />
+
+## Request a change with a Jira issue
+
+If you would rather not [contribute to the documentation yourself](#contribute-to-documentation-yourself) let us know about the issue by adding a Jira ticket. Use the following procedure to create a Jira ticket:
+
+In the Contact Us section, select the link in "open a JIRA ticket" 
+
+<img src = "./docs/_images/jira-ticket.png" alt="JIRA ticket" width = 30% height = 30% />
+
+- Select the **open a JIRA ticket** link in the Contact Us section on the bottom of the page. This action opens the [Jira issue tracker](https://jira.percona.com/projects/PSQLADM/issues) for the doc project.
+
+- Log in (create a Jira account if you don't have one) and select **Create** to open the Jira form.
+  
+
+<img src = "./docs/_images/create-issue.png" alt="Complete the Jira form" width = 30% height = 30% />
+
+- In the following fields, describe the issue:
+
+  - In the Summary, provide a brief description of the issue.
+
+  - In the Description, provide more information about the issue. If needed, add a Steps To Reproduce section and information about your environment (version number, your operating system, etc.). Be detailed. 
+  
+  - Select **CREATE** to create the ticket.
+
+## Contribute to documentation yourself
+
+Using either the [Edit documentation online with GitHub](#edit-documentation-online-with-github) method or the [Edit documentation locally](#edit-documentation-locally) method to make changes to the documentation and create a pull request. 
+ 
+
+### What you should know
+
+Most of the document is in plain text, but you may use [Markdown] (https://www.markdownguide.org/) to add syntax elements (notes, tables, and so on) to the documentation. 
+
+### What happens after you create the pull request
+
+A core developer reviews your request and either comments or approves the request. A core developer merges the approved request to the **main** branch usually when a newer version of the product is released.
+
+!!! note
+
+    We appreciate your work but the pull request may be redone to meet internal requirements.
+
+### Edit documentation online with GitHub
+
+In the upper right corner, select the pencil icon to open the source file in the GitHub editor. In this editor, you can make the changes, view the changes in the Preview tab, and create a pull request. The source `.md` file of the page opens in a GitHub editor in your browser. If you haven't worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+Edit the page using the Markdown syntax.
```

Comment:
Should we add here a link where the user can check Markdown syntax?

Reply:
Added.